src/gui/lineedit/src/lineedit.h

```diff
@@ -12,26 +12,29 @@
 
 #include <QLineEdit>
 
-QT_BEGIN_NAMESPACE
 class QToolButton;
-QT_END_NAMESPACE
 
 class LineEdit : public QLineEdit
 {
   Q_OBJECT
 
 public:
-  LineEdit(QWidget *parent = 0);
```

Comment:
>I'll change it back if no further discussion later.

You can leave it as is. I understand your reasons before doing this discussion.
It's just thoughts on a related topic...